tests/models/paligemma/test_modeling_paligemma.py

```diff
@@ -340,6 +340,32 @@ def test_small_model_integration_test(self):
             EXPECTED_DECODED_TEXT,
         )
 
+    @slow
+    # @require_read_token
```

Comment:
same here

Reply:
yes, wanted to leave on general class level and remove from each test, seems reduntant otherwise